src/App.vue

```diff
@@ -40,8 +40,11 @@ export default defineComponent({
 
 		// Fetch authed user
 		const { onResult } = useQuery<GetUser>(GetUser, { id: "@me" });
-		onResult(({ data: { user } }) => {
-			store.commit("SET_USER", user);
+		onResult((res) => {
+			if (!res.data) {
```

Comment:
if they are not logged in?

Reply:
Just temporary until I fully figure out auth